readers-frontend/src/pages/articles/[year]/[month]/[day]/[slug].astro

```diff
@@ -202,7 +202,7 @@ const imagePlaceHolder =
           <div
             id="article-content"
             class="whitespace-pre-wrap text-justify"
-            set:html={articleDraft.articlesDrafts_text}
+            set:html={articleDraft.articlesDrafts_text + "<style scoped>div > p:first-child:first-letter {float: left;font-size: 90px;line-height: 60px;padding-top: 4px;padding-right: 8px;padding-left: 3px;}</style>"}
```

Comment:
Don't concatenate the raw article html with a style tag. It'd be more elegant to follow the Astro style pattern. You can add the style element as you would in a static HTML page. See [the Astro docs](https://docs.astro.build/en/guides/styling/) and [this example](https://github.com/yorknouse/website/blob/c5911e2493a0965581136c9308e6d61931bed138/readers-frontend/src/components/Article.astro#L42).

However, the element should be conditionally rendered depending on whether the article being rendered  [wants dropcaps or not](https://github.com/yorknouse/website/blob/master/readers-frontend/prisma/schema.prisma#L71). I.e. :

```jsx
{
    article.articles_dropCapital && (
        <style>
        ...
        </style>
    )
}
```